### PR TITLE
Added some helpful comments to Linux setup for newbies.

### PR DESCRIPTION
**Pull request by Hwurzburg**
a little more detail to help linux/sphinx/python newbies when setting up the environment in a native Linux machine (which caused me some issues  when installing it)

### CODE REVIEW

File: common/source/docs/common-wiki-editing-setup.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Before submitting large changes to the wiki it is best to check the pages are re
 There are two methods to setup a local build environment:
 
 #. Use the `Vagrantfile <https://github.com/ArduPilot/ardupilot_wiki/blob/master/Vagrantfile>`__ in the root of the repo to create a Linux virtual machine with all the necessary packages installed.  This is the preferred and supported method
-#. Install `Sphinx <http://www.sphinx-doc.org/en/stable/install.html>`__ on your local Linux machine and skip the Vagrant instructions below (unsupported). To do this you can simply edit the Vagrantfile removing the specific vagrant provisioning commands, and run the Vagrantfile as a shell script.
+#. Install `Sphinx <http://www.sphinx-doc.org/en/stable/install.html>`__ on your local Linux machine and skip the Vagrant instructions below (unsupported). To do this you can simply edit the Vagrantfile removing the specific vagrant provisioning commands (remove lines before "apt-get -y ..."), and run the Vagrantfile as a shell (.sh)script. Be sure to use the sudo prefrix. Then jump down to "Build the wiki".
```
Comment (khancyr) [inline common/source/docs/common-wiki-editing-setup.rst]: Prefix

File: common/source/docs/common-wiki-editing-setup.rst [hunk 2]
```diff
@@ -64,7 +64,7 @@ The main steps for building the docs are:
 Build the Wiki
 --------------
 
-As shown in the last step of the vagrant instructions above, use update.py to build some or all of the wiki
+As shown in the last step of the vagrant instructions above, use update.py to build some or all of the wiki. You may need to prefix the python commands below with " sudo ...", especially when rebuilding.
```
Comment (khancyr) [inline common/source/docs/common-wiki-editing-setup.rst]: Just sudo on the prefix ! Let's not confuse them with ... ;-p

File: common/source/docs/common-wiki-editing-setup.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-wiki-editing-setup.rst]: tnx

File: common/source/docs/common-wiki-editing-setup.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-wiki-editing-setup.rst]: done

**Pull request merged**